### ISSUE TITLE
[EN][Talent][Rogue] Venomous Wounds

### ISSUE THREAD

**Issue by Aldurion** [Atlantiss/BugTracker#4191]
The damage shown on Venomous wounds talent page is not the same with the damage from target dummy. 
![wowscrnshot_080215_040027](https://cloud.githubusercontent.com/assets/3969232/9194001/c05657ea-4020-11e5-8102-268d7289fbbb.jpg)

The talent page mentions 4163 damage but on target dummy the poison damage form Venomous wounds is only 3473.

I  noticed this closed issue about damage not scaling while leveling but it seems it is not scaled at lv 85 either. https://github.com/Atlantiss/BugTracker/issues/3400

There were also other closed issues regarding Venomous Wounds but hell if I know what it is written in them. Google translate is not helpful in translating/learning Polish...

**Comment by planche** [comment]
you can do the maths by yourself 
" Venomous Wounds base damage is given by (363+.135*AP) at level 80 and (675+.176*AP) at level 85. It crits at your usual spell crit rate, and benefits from Potent Poisons and the spell damage buff (Master Poisoner or equivalent) but not Vile Poisons."
http://forums.elitistjerks.com/topic/104491-cataclysm-mechanics-testing/

**Comment by Aldurion** [comment]
"but not Vile Poisons", yeah, apparently so...

**Comment by Jenovas** [comment]
Please calculate the DMG, Tooltips arent accurate.

**Comment by Aldurion** [comment]
OK, did a new test. ( 675 + 0.176* 9560 melee attack power)* 1.08 (master poisoner) *1.64 ( mastery rating 1880 which is mastery 18.49 which is 64% more damage with poisons) =2356,56*1.08*1.64= 4175?!

![wowscrnshot_081115_155918](https://cloud.githubusercontent.com/assets/3969232/9198065/fae7abae-4041-11e5-99d0-353eb83c0eb8.jpg)
but here i get 3168 poison damage and tooltip says 3866, am I missing something?

**Comment by Aldurion** [comment]
I did a new test, (9560 melee attack power*0.176+ 675 ) * 1.08 (master poisoner) * 1.64 (18.49 mastery which gives 64% more damage with poisons) = 2357.56 * 1.08 * 1.64= 4175.71 .
But here I get 3168 while the tooltip says 3866, what am I missing?
![wowscrnshot_081115_160700](https://cloud.githubusercontent.com/assets/3969232/9198280/bc9cd7f0-4043-11e5-97be-0b1871f7b05a.jpg)

**Comment by planche** [comment]
Well that's weird it was fine few weeks ago.

**Comment by Lenox07** [comment]
There is something wrong with base damage or scaling.
On naked character (771 ap and 28% bonus from mastery) vw did 177 dmg where it should do about 1120 (tooltip dmg is 1090).
![wowscrnshot_081115_183138](https://cloud.githubusercontent.com/assets/9517740/9204092/5e5a7450-405b-11e5-83b4-1df31b2a571f.jpg)
On geared character (11367 ap and 76% bonus from mastery) vw did 4550 dmg where it should do 5084 (tooltip dmg is 4944).
![wowscrnshot_081115_190639](https://cloud.githubusercontent.com/assets/9517740/9204229/32ded02c-405c-11e5-957d-9245b9bb3e88.jpg)

**Comment by tnarf** [comment]
Problem is in talent Vile Poisons (VP).

build without Master Poisoner; target skull dummy
ap: 9564
mastery: 65%

tooltip and formula dmg: 3891
dmg with
0 points in VP: 4630
1 points in VP: 4074
2 points in VP: 3519
3 points in VP: 3200

**Comment by Jenovas** [comment]
Should be fixed in rev 5015

**Comment by Prodiege** [comment]
So is the change essentially a damage buff to assassination rogue now? 

I was confused reading the changelog: Quoted saying "Venomous Wounds DMG should no longer be lowered by Vile Poisons". Was Vile Poisons reducing the Venomous Wounds damage by 36%? Now does Vile Poisons increase its damage by 36% or just not reduce it?

Also it said quoted "Venomous Wounds AP coef has be corrected (lowered)". How much was the AP coefficient changed? 

Basically: is this an overall dps increase change, static, or dps loss?

**Comment by Jenovas** [comment]
Increase
